salt/_pillar/metalk8s.py

```diff
@@ -39,28 +39,6 @@ def _load_networks(config_data):
     }
 
 
-def _get_labels(config="/etc/kubernetes/admin.conf",
-                context="kubernetes-admin@kubernetes"):
-    # TODO: Use the "master kubeconfig" instead of /etc/kubernetes/admin.conf
-    #       (Refs: #691)
-    labels = {}
-
-    hostname = __grains__.get('localhost')
-    if not hostname or not os.path.isfile(config):
-        return labels
-
-    try:
-        labels = __salt__['kubernetes.node_labels'](
-            name=hostname,
-            kubeconfig=config,
-            context=context
-        )
-    except Exception as exc:  # pylint: disable=broad-except
-        log.error('Unable to get kubernetes labels for %s:\n%s', hostname, exc)
-
-    return labels
-
-
```

Comment:
You need to remove this also in the `ext_pillar` function just below

Reply:
Yeah, ran into that while testing :P But wanted early feedback. Expect a push fixing this.